examples/common/metrics_foo_library/foo_library.cc

```diff
@@ -34,8 +37,6 @@ class MeasurementFetcher
 public:
   static void Fetcher(opentelemetry::metrics::ObserverResult observer_result, void * /* state */)
   {
-    std::map<std::string, std::string> labels = get_random_attr();
```

Comment:
Hi @lalitb, I still have race by reverting these two line that are not used. This suggest we have timing issue somewhere :(

Reply:
please ignore, seems the issue is related to the `get_random_attr` itself.